pkg/apis/config/scheme/scheme.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 The Multicluster-Scheduler Authors.
```

Comment:
thoughts on dropping the year from the boilerplate header?

Reply:
You can update it in https://github.com/admiraltyio/admiralty/blob/master/hack/boilerplate.go.txt